spec/controllers/two_factor_authentication/otp_verification_controller_spec.rb

```diff
@@ -122,41 +122,43 @@
   end
 
   describe '#create' do
-    let(:parsed_phone) { Phonelib.parse(controller.current_user.default_phone_configuration.phone) }
+    let(:user) { create(:user, :with_phone) }
```

Comment:
The reason for this change is that `have_logged_event` coming after the controller action works differently than how analytics was previously stubbed with `expect` _before_ the action, and certain properties rely on the state of the user before the action. Some values like `controller.current_user` are unavailable after the action is run.